pontoon/base/errors.py

```diff
@@ -0,0 +1,10 @@
+from raygun4py import raygunprovider
+
+from django.conf import settings
+
+def send_exception():
+    """
+    Function sends exception to selected provider. This will allow us to change
```

Comment:
Explaining that this allows us to abstract away which specific error service we're using isn't necessary, leave it out. Same with the mention of Raygun; the developer can already see that we're importing Raygun libs, they don't need to be told twice.